blog/2024/keyconf24-invitation.adoc

```diff
@@ -0,0 +1,49 @@
+:title: Announcing Keycloak's Identity Summit: KeyConf24
+:date: 2024-06-10
+:publish: true
+:author: Nathalia Pinesi
+
+KeyConf23 was an incredible success, bringing together nearly 60 passionate members of the Keycloak community in London. The energy and collaboration were palpable as attendees
+delved into the latest developments in identity and access management. We witnessed thought-provoking discussions, learned from industry experts, and forged valuable connections.
+
+Building on that momentum, we're thrilled to announce KeyConf24, our 2024 Keycloak Identity Summit! This year's event promises to be even bigger and better, with a program packed
+full of relevant, cutting-edge topics.
+
+== Save the Date and Join Us!
+
+We invite all developers, architects, security professionals, and anyone interested in identity and access management to join us for KeyConf24.
+The event will happen in Vienna, Austria on 19th September 2024. Please register on https://www.eventbrite.de/e/keyconf24-tickets-887467387847[this form] for free if you want to
+join us.
+
+In the meantime, mark your calendars and prepare for an unforgettable experience!
+
+== What to Expect at KeyConf24
+
+* *European Digital Identity Wallet:* Deep dives into the European Union's ambitious initiative and its impact on identity management.
+
+* *Verifiable Credentials:* Explore the exciting potential of decentralized identity verification and the role of Keycloak.
+
+* *State of OIDC FAPI2:* Get the latest updates on OpenID Connect's Financial-grade API (FAPI) security profile.
+
+* *New Grant Type SPI & Token Exchange Endpoints:* Technical sessions on Keycloak's expanded capabilities and how to leverage them.
+
+* *Many more to be announced...*
+
+And of course, there will be ample opportunities for networking, knowledge sharing, and connecting with the vibrant Keycloak community and Keycloak maintainers.
+
+== Save the Date and Join Us!
```

Comment:
It seems that L12-18 and L34-38 are the same sentences.

Reply:
Thanks Takashi! It was not exactly same, but little bit similar. The lines 12-18 were outdated, so removed them from the PR.